src/v1/components/counters/countersController.js

```diff
@@ -98,45 +98,42 @@ class Counters {
    */
   //
   async getDashboardAdminCounters (_, response, next) {
-    try {    
+    try {
       // Total approved contributions
       const totalApprovedContributionsRequest = `
         SELECT COUNT(*)
         FROM "references"
         INNER JOIN "users" ON "users".id = "references".reference_moderator_id
         WHERE "references".reference_status = true;
       `;
-      const totalApprovedContributionsResult = await Postgres.query(totalPendingContributionsRequest);
-    
+      const totalApprovedContributionsResult = await Postgres.query(totalApprovedContributionsRequest);
       // Total pending contributions
       const totalPendingContributionsRequest = `
         SELECT COUNT(*)
         FROM "references"
         WHERE "references".reference_status = false;
       `;
-      const pendingContributions = await Postgres.query(totalPendingContributionsRequest);
-    
+      const totalPendingContributionsResult = await Postgres.query(totalPendingContributionsRequest);
       // Total contributors
       const totalContributorsRequest = `
         SELECT COUNT(DISTINCT "reference_contributor_id")
         FROM "references"
         WHERE "reference_status" = true
       ;`;
```

Comment:
Merge all these 3 requests in 1 to save DB calls since we're getting infos from the same table